do_task.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+import yaml
+
+logger = logging.getLogger(__name__)
```

Comment:
We might consider adding something to show how to set up a custom logger. The default logger level is `WARNING`, so the calls below to `INFO`, which is a level below `WARNING` won't print to screen. Not sure if that's the intended behavior.

Reply:
Nope - I've never actually used a logger myself 😀.  Perhaps we use the verbose example in the `parser` to set the logger level?

Could be a fix here, or a separate PR later - make an issue?